### PR TITLE
Expose agent persistence for server users

### PR DESCRIPTION
**Pull request by joshmeek**
PR Checklist:

- [x] add a short description of what's changed to the top of the `CHANGELOG.md`
- [x] add/update tests (or don't, for reasons explained below)

## Describe this PR
This PR exposes the agents tile and page for users of server. It also improves some of the information displayed on the cards in the agent page. Submitted flow runs will now display the ID of the agent that submitted them and links back to the agent page.

The agents page will now function much better after https://github.com/PrefectHQ/server/pull/95 is released.

@zhen0 @znicholasbrown Do you usually include screenshots in PRs? I can attach some if needed

### CODE REVIEW

File: src/graphql/FlowRun/flow-run.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ export default function(isCloud) {
         state_timestamp
         parameters
         context
+        agent_id
```
Comment (zhen0) [inline src/graphql/FlowRun/flow-run.js:38]: This is in the schema for flow_run but it's currently breaking the flow run page in cloud.  Error message is: "GraphQL error: field \"agent_id\" not found in type: 'flow_run'" We could query for agent {id} here but probably better that we check Cloud?

File: package-lock.json [hunk 2]
```diff
@@ -5090,6 +5090,17 @@
             "unique-filename": "^1.1.1"
           }
         },
+        "chalk": {
```
Comment (zhen0) [inline package-lock.json:5093]: Are you meaning to add these?

**Review by zhen0 — CHANGES_REQUESTED** [review]
Thanks @joshmeek!  Very cool to have a new member of the UI team 😉 and it'll be great to have agents in server.  

A few suggested changes but my main concern is that agent_id seems to be breaking the Flow Run page in Cloud (tried in dev and staging).

File: src/graphql/FlowRun/flow-run.js (hunk 1 above)
Comment (cicdw) [reply src/graphql/FlowRun/flow-run.js:38]: Hm that sounds like a bug in our Hasura metadata in Cloud. I'll take a look and report back

File: package-lock.json (hunk 2 above)
Comment (joshmeek) [reply package-lock.json:5093]: No clue, this was updated when I ran `npm install` I think 🤔

File: src/graphql/Agent/delete-agent.gql [hunk 3]
```diff
@@ -1,4 +1,4 @@
-mutation DeleteAgent($agentId: String!) {
+mutation DeleteAgent($agentId: UUID!) {
```
[the remark this answers is not in the document]
Comment (joshmeek) [reply src/graphql/Agent/delete-agent.gql:1]: I believe this should be resolved now 👍

File: src/components/Agents/AgentCard.vue [hunk 4]
```diff
@@ -312,13 +315,48 @@ export default {
...
+            CREATED
+          </div>
+          <div class="body-2">
+            {{ formDate(agent.created) || 'Unknown' }}
+          </div>
+        </div>
+
+        <div>
+          <div class="caption">
+            Agent ID
```
Comment (zhen0) [inline src/components/Agents/AgentCard.vue:329]:
```suggestion
            AGENT ID
```

**Review by zhen0 — COMMENTED** [review]
One small suggestion then looks good to me @joshmeek. I'll let @znicholasbrown do a final review.

**Comment by znicholasbrown** [comment]
Hm, I'm unable to visit flow run pages with this, getting:
```
"field "agent_id" not found in type: 'flow_run'"
```
Getting the same with both Server and Cloud (prod). Should I be testing this in a different environment? It looks like agent stuff was released with Server 4 days ago.

**Comment by cicdw** [comment]
@znicholasbrown that was a minor bug, I'm about to release the fix in one minute.

**Review by znicholasbrown — APPROVED** [review]
LGTM @joshmeek 🎉

**Pull request merged**